database/migrations/default/1765230748998_crash_address_display_trigger/up.sql

```diff
@@ -0,0 +1,776 @@
+ALTER TABLE crashes ADD COLUMN address_display TEXT;
+
+-- Create an index on the new column
+CREATE INDEX idx_crashes_address_display ON crashes(address_display);
+
+-- Function to format highway addresses
+CREATE OR REPLACE FUNCTION format_highway_address(
```

Comment:
I love having these two separate functions for highways vs streets. Really easy to follow what's happening 🚀